api/app/Http/Controllers/UserController.php

```diff
@@ -177,6 +177,12 @@ public function patchOne(Request $request, $id)
         // Extract the credentials and update if necessary
         $credentialUpdateDetails = $request->get('_user_credential', []);
         if (!empty($credentialUpdateDetails)) {
+            // Invalidate token for the user when user changes their password
+            if ($this->jwtAuth->user()->user_id == $model->user_id) {
```

Comment:
Ah actually I get it now, it's for when an admin changes a users password right?